internal/app/orchestrator/orchestrator.go

```diff
@@ -249,6 +251,17 @@ func (o *orchestrator) GetDownstreamAggregatedKeys() (map[string]bool, error) {
 	return keys, err
 }
 
+func (o *orchestrator) ClearCacheEntries(keys []string) []error {
+	var errors []error
+	for _, key := range keys {
+		_, err := o.cache.DeleteKey(key)
```

Comment:
this will remove the entry from the cache, does it also terminate the sidecar streams?

Reply:
`DeleteKey()` internally calls `c.cache.Remove(key)` [here](https://github.com/envoyproxy/xds-relay/blob/1c332a20076edbbd5b6250c6c90cea342a5683a9/internal/app/cache/cache.go#L260). The intenral lru Remove() call both removes the element and runs the `onEvicted` func as seen [here](https://github.com/golang/groupcache/blob/master/lru/lru.go#L86-L113). Because when the cache was created, we passed [onEvictCache](https://github.com/envoyproxy/xds-relay/blob/master/internal/app/orchestrator/orchestrator.go#L101) to it, it should clean up any related streams.